Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1793,6 +1797,11 @@ def tableToMarkdown(name, t, headers=None, headerTransform=None, removeNull=Fals
         headers = list(t[0].keys())
         headers.sort()
 
+    if first_headers:
+        for h in reversed(first_headers):
+            if h in headers:
+                headers.insert(0, headers.pop(headers.index(h)))
+
```

Comment:
I like the code, but I'm not sure we need this given we have `headers`. I realize that it's easier to "blindly" select the first headers, while not caring if they exist or what the other headers are.
@yuvalbenshalom wdyt?

Please add UTs.

Reply:
Why is that needed in general?

Reply:
This is following the request to have some columns appear before others, from the demo session.

Given the beta-ness of the API I preferred writing this generic mechanism, which could also be used in similar cases in the future.

Other possibilities would be locally sorting the headers (copying this snippet to Azure's code), or just hardcoding the headers.

Reply:
since this is not a common need in other integrations(as far as I know)
and it is bit confusing(first_headers/headers), I am not sure it Weill be a good general addition.
I think it is better to do in the integration.

Reply:
Alright, reverted CSP changes, sorting happens locally on AzureAD.py